addons/sup_combatsupport/scripts/NEO_radio/functions/ui/transport/fn_transportTaskLbSelChanged.sqf

```diff
@@ -100,7 +100,7 @@ _task = switch (_lb lbText _index) do
 
         _pos = getMarkerPos (uinamespace getVariable ["NEO_transportMarkerCreated","unknown"]);
 
-        if (isNil "_pos" || {str(_pos) == "[0,0,0]"}) then {_pos = getpos player;};
+        if (isNil "_pos" || {str(_pos) isEqualTo "[0,0,0]"}) then {_pos = getpos player;};
```

Comment:
the to string is useless.
`_pos isEqualTo [0,0,0]`

Reply:
Also I don't see how _pos could EVER be nil. getVariable cannot return a nil because it has a default value. Thus getMarkerPos also can never return nil.

Reply:
Replaced